scripts/setup.py

```diff
@@ -0,0 +1,57 @@
+import shutil
```

Comment:
One concern I have is this now requires Python to be installed. Our current process is Python-independent until now (consider most Android developers won't be running the backend server, and many won't be using Ubuntu that often comes with Python preinstalled).

Two alternative options
1. Use Kotlin with Bazel & bash (bash would just run a Bazel command to run the script where the script is written in Kotli)
2. Use native Bash

The former may be more work, but's probably more future-proof as the setup script gets more complex.

@vinitamurthi @rt4914 any thoughts?

Reply:
With native bash, we can have a simple script of maybe 2 line for moving/copying of pre commit file.

Reply:
I think Bash can be a good option. As we have more things like this we can integrate that in bash file.

Reply:
Done